gtk/po/POTFILES.in

```diff
@@ -1,5 +1,6 @@
 # List of source files containing translatable strings.
 src/ghb3.ui
+src/ghb4.ui
```

Comment:
This shouldn't be needed. ght4.ui isn't used yet. It is part of my migration to gtk4 which I haven't updated in quite some time. It's probably time to make a more serious effort of completing that migration, but gtk4 was still in quite a lot of flux at the time I started this.

Reply:
I see. intltool-update told me it was missing. I will remove it then.